.clang-format

```diff
@@ -0,0 +1,136 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Microsoft
```

Comment:
Why not just Microsoft + useTabs ?

Reply:
It can be, I dump the config from command line then changed the useTabs, so all of configuration is there. If we want to make it simpler I can change it

Reply:
If you made this dump with clang-format-13 there are some params that would be not available on clang-format-10 for example. 

I think just Microsoft + useTabs Is the easiest to read and if we need a specific change we can explicit it no ?